### DISCUSSION TITLE
AWS EKS Cluster Backup: PV and PVC Restoration Across Regions

### DISCUSSION THREAD

**Discussion by chandakanant** [vmware-tanzu/velero#8359] [Community support Q&A]
I have an AWS EKS cluster deployed in the **Ireland** region (eu-west-1) and am looking to perform a complete backup of the cluster, including persistent volumes (PV) and persistent volume claims (PVC). My question is whether I can successfully restore this backup in the **London** region (eu-west-2) without any issues related to the PV and PVC, or any other issue I should keep in mind.

**Comment by kaovilai** [comment]
I don't think there should be any issues. You may want to use https://velero.io/docs/v1.15/csi-snapshot-data-movement/ to move data across regions.

**Reply by chandakanant** [reply]
@kaovilai how do I do it using helm chart/values.yaml?

**Reply by kaovilai** [reply]
[Set this](https://github.com/vmware-tanzu/helm-charts/blob/bbb36ccf357d6baf7f13accbc0b169e6efceb3d4/charts/velero/values.yaml#L438-L439) I assume.
Otherwise read docs above.
